test/Microsoft.AspNetCore.Server.KestrelTests/HttpsConnectionFilterTests.cs

```diff
@@ -22,6 +23,13 @@ namespace Microsoft.AspNetCore.Server.KestrelTests
 {
     public class HttpsConnectionFilterTests
     {
+#if NET451
+        static HttpsConnectionFilterTests()
+        {
+            ServicePointManager.SecurityProtocol |= SecurityProtocolType.Tls12 | SecurityProtocolType.Tls11;
```

Comment:
Actually I need to revisit that. I added this code when I restricted it to TLS 1.2 (which needs to be explicitly enabled), but since we're allowing TLS 1.1 too this might not be needed.